### ISSUE TITLE
Accessibility - unable to select options when VoiceOver (iPhone 6) is on

### ISSUE THREAD

**Issue by markjoeljimenez** [jshjohnson/Choices#555]
When VoiceOver is turned on in **iPhone 6**, the user is unable to select an option. Instead Choices reads out the option then closes immediately.

### Steps to reproduce:

1. Turn on VoiceOver
2. Go to a single-select Choice
3. Choose an option

### Expected behaviour
Selects the option

### Actual behaviour
VoiceOver reads out option then closes dropdown immediately before I can double tap to select the option

### Video Example
iPhones don't record what VoiceOver says when screen recording so it's a little difficult to show what's happening. 

[Gfycat](https://gfycat.com/unselfishrectangularbengaltiger)

When I open up the dropdown, I single-tapped 'Choice 2' to focus on it. The dropdown closes immediately before I can double-tap to select. VoiceOver then focuses on the element behind the 'Choice 2' option which is the 'Fake Tales of San Francisco' select.
